indexstyle.css

```diff
@@ -233,6 +250,17 @@ footer {
 } 
```

Comment:
I'd probably increase the size of your social Icons a little, they're kinda small at the minute.

Reply:
Done! I was using an old stylesheet. Sorted now.